doc/api/child_process.md

```diff
@@ -395,6 +397,14 @@ child.on('error', (err) => {
 });
 ```
 
+*Note: Certain platforms (OS X, Linux) will use the value of `argv[0]` for the
```

Comment:
Every other "Note" in the file currently italicizes the whole note ( https://github.com/nodejs/node/blame/master/doc/api/child_process.md#L26, https://github.com/nodejs/node/blame/master/doc/api/child_process.md#L39,  https://github.com/nodejs/node/blame/master/doc/api/child_process.md#L188 etc). I'm happy to change this one instance if you want, but should it really be inconsistent with the rest of the file?